src/content/docs/reference/builders.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Builders
+description: A reference detailing how to use Dev Containers and custom images as the base for your project.
+sidebar:
+  label: Builders
+---
+import Aside from '@components/Aside.astro'
+
+Builders are responsible for creating a container image with everything you need to be productive.
+Daytona allows for control over how the resulting Project environment is created, allowing you to choose your preferred method to pull in your project's tools, dependencies, and configuration.
+
+At Workspace creation time, you can choose between the following options:
+
+1.  __Automatic__  
+    Daytona builds the Project image after auto-detecting the most appropriate Builder.
+2.  [__Dev Container__](#dev-container)  
+    Daytona builds the Project image according to the Dev Container configuration located in the project.
+3.  [__Custom Image__](#custom-image)  
+    Daytona builds the Project image using an image you specify.
+4.  __None__  
+    Daytona creates the Project using the default base image (`daytonaio/workspace-project`).
+
+## Supported Builders
+### Dev Container
+You can build the Project image using the Dev Container standard.
+The following options are accepted when using this Builder:
+
+1.  __Devcontainer file path__  
+    The path where the Dev Container configuration is located, relative to the repository root.
```

Comment:
I did understand where I input this root?

Reply:
@osslate I agree that we should mention that choosing builders is available through the advanced configuration menu.

I see that you mentioned it in the Workspaces page but it might be worthwhile to mention it on this page as well.

Reply:
Clarified in the introduction ([54d4e99](https://github.com/daytonaio/docs/pull/56/commits/54d4e991cfba5a4d50648326b5e28850e28a90e9))